docs/usermanual.md

```diff
@@ -98,17 +98,50 @@ hello
 
 then the resulting attribute sent to ContextBroker:
 
-{
-   ...
-   "attrHex": {
-     "value": "68656c6c6f"
-     "type": "<the one used at provisiong time for attrHex attribute>"
-   }
+{ ... "attrHex": { "value": "68656c6c6f" "type": "<the one used at provisiong time for attrHex attribute>" } }
+
+Note that every group of 2 character (I.E, the first group, `68`) corresponds to a single ASCII character or byte
+received in the payload (in this case, the value `0x68` corresponds to `h` in ASCII). You can use one of the multiple
+tools available online like [this one](https://string-functions.com/string-hex.aspx)
+
+##### NGSI-v2 and NGSI-LD Measure reporting
+
+It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with payloadType
+`ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure attributes and
+`id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used.
+
+Examples of these ngsi payloads are:
+
+```
+ {
+     "actionType": "append",
+     "entities": [
+        {
+          "id": "MyEntityId1",
+          "type": "MyEntityType1",
+          "attr1": { type: "Text", "value": "MyAttr1Value"},
+          ...
+        },
+        ...
+    ]
 }
+```
 
-Note that every group of 2 character (I.E, the first group, `68`) corresponds to a single ASCII character or byte received in 
-the payload (in this case, the value `0x68` corresponds to `h` in ASCII). You can use one of the multiple tools available 
-online like [this one](https://string-functions.com/string-hex.aspx)
+and
+
+```
+ [
+    {
+          id: 'MyEntityId1',
+          type: 'MyEntityType1',
+          attr1: { type: 'text', value: 'MyAttr1Value'},
+          ...
+    },
+    ...
+]
+```
+
```

Comment:
As discussed at skype, it would be great to include a third example with a NGSI-LD example, for completeness.

Reply:
done in https://github.com/telefonicaid/iotagent-json/pull/779/commits/6c93ffe423d991bcec222792dc958c951e8ed0b3